webapp/src/components/kanban/__snapshots__/kanban.test.tsx.snap

```diff
@@ -279,6 +279,7 @@ exports[`src/component/kanban/kanban return kanban and change title on KanbanCol
             </span>
           </div>
           <button
+            title="limited-card-count"
```

Comment:
Based on the code changes, nothing should change since your checking 
`if (limitedCard.length > 0) {`
and there are no changes to test data.
In addition, this item is `<span>0</span` which shouldn't happen.

Reply:
Realize now, the 0 is because its a "Hidden Column" not "limited-card-count"